cloudinit/net/network_state.py

```diff
@@ -725,7 +725,7 @@ def _handle_bond_bridge(self, command, cmd_type=None):
             # we accept the fixed spelling, but write the old for compatability
             # Xenial does not have an updated netplan which supports the
             # correct spelling.  LP: #1756701
-            params = item_params['parameters']
+            params = item_params.get('parameters', {})
```

Comment:
[On line 738](https://github.com/candlerb/cloud-init/blob/bridge-parameters/cloudinit/net/network_state.py#L738), there is also a `item_params.get('parameters', {})`. Can you replace it with the `params` defined here to avoid confusion?

Reply:
Done